utils/getOffchainPrice.ts

```diff
@@ -30,6 +30,19 @@ function _getBitstampPriceFromJSON(jsonData: any) {
   return Number(jsonData.last);
 }
 
+// This is needed because our new cors proxy wont forward http or https. We really only use https.
+function stripProtocol(url: string) {
+  return url.replace("https://", "");
+}
+
+// Small helper to wrap a url in cors proxy.
+function proxyUrl(
+  url: string,
+  proxy = "https://us-central1-uma-protocol.cloudfunctions.net/cors-proxy"
```

Comment:
Long-term, we should probably have this proxy address passed in as a build env variable.

Reply:
not sure if this project will reference a .env, didnt try. If it does though it should be easy to add

Reply:
added in optional env var. see comment